cmd/user/internal/application/eventhandler/when_user_was_registered_with_email.go

```diff
@@ -57,11 +57,21 @@ func (u userWasRegisteredWithEmailModel) GetID() string {
 	return u.e.ID.String()
 }
 
+// GetName the full name
+func (u userWasRegisteredWithEmailModel) GetName() string {
```

Comment:
why `*Model` ?

Reply:
I just reuse it the same way it's been used throughout the file.

Reply:
> I just reuse it the same way it's been used throughout the file.
I dont think i understand

we do not have events such as userWasRegisteredWithEmailModel (with `Model` suffix) and if we do the name is invalid